recipes/xgboost/build.sh

```diff
@@ -33,7 +33,7 @@ cmake \
       -D USE_CUDF:BOOL=ON \
       -D CUDF_ROOT:PATH="${PREFIX}" \
       -D CUDF_INCLUDE_DIR:PATH="${PREFIX}/include" \
-      -D GPU_COMPUTE_VER:STRING="60;61;70" \
+      -D GPU_COMPUTE_VER:STRING="60;61;70;80" \
```

Comment:
Should we match cudf? Meaning drop 61 and add 75 & 80 (https://github.com/rapidsai/cudf/blob/branch-0.17/cpp/CMakeLists.txt#L97-L105)

Reply:
Good catch